backend/etl/soft_story_properties_data_handler.py

```diff
@@ -65,12 +61,17 @@ def parse_data(self, data: dict) -> list[dict]:
             geometry = feature.get("geometry", {})
             if geometry:
                 geom_longitude, geom_latitude = geometry["coordinates"]
-            addresses.append(properties.get("address"))
 
             # Search for the address in the mapbox geojson
-            mapbox_coordinates = self.mapbox_geojson_manager.get_mapbox_coordinates(
+            if not self.mapbox_geojson_manager.is_address_in_geojson(
```

Comment:
If not found in the geojson, store to fill in later -- this is reserved for new addresses in the SFData dataset.